includes/page_top.php

```diff
@@ -2,7 +2,7 @@
     require_once $APPLICATION_DIR.'/includes/application.php';
     $page_request = $_GET['page_request'] ?? \FirstAide\Router::INDEX;
     $query = $_GET['query'] ?? '';
-    $page = FirstAide\Router::getPage($page_request, $query);
+    $page = FirstAide\Router::getPage($UserObj, $page_request, $query);
```

Comment:
Can you explain what has changed here and why? You can start adding comments on your PR  explaining changes if there are no comments in that file